### ISSUE TITLE
App immediately errors on mixed-generation-GPU systems with a CC3.0 GPU

### ISSUE THREAD

**Issue by tonigi** [Acellera/acemd_issues#43]
(From BOINC users.)

From: Jacob Klein (ID 8048)
Subject: AceMD - Update?

1 GTX 970 GPU, 2 GTX 660 Ti GPUs:
[url]https://www.gpugrid.net/results.php?hostid=153764[/url]

**Comment by j3mdamas** [comment]
@tonigi I don't know about boinc, so can you help me figure out what is the issue here?

**Comment by tonigi** [comment]
It's a report from a cruncher. The link (fixed: https://www.gpugrid.net/result.php?resultid=16703390 )  provides info on the system.

I thought it was unrelated to boinc, ie. it occurred even in the "non boinc" case, but I DO have a 680 + 2 x 1080, and the "normal" acemd does  work.

So may be something windows specific... ouch.

T

**Comment by j3mdamas** [comment]
I only have compiled versions for Linux, I never compiled for Windows. I am going to have a meeting with Matt tomorrow, I'll ask him about this.

**Comment by tonigi** [comment]
More details here... I can help of course, but we need to clarify a bit how to build, what version is on, etc.

----

Thanks. Both issues were introduced with v9.18 of the app, and both were reported at the time, and neither has been fixed.

For several months (since April 2017?), I've watched the tasks error on my CC3.0 GTX 660 Ti GPUs, and for several months I've had to wait 5-30 seconds when suspending BOINC, to wait for acemd to spool down. My other 15 active projects do not have the issue.

Those are the 2 issues that ... stand out as obviously needing some work, to make the GPUGrid project run smoother, to me.

[url]http://www.gpugrid.net/forum_thread.php?id=4551[/url]

If you would like me to test anything, and can provide clear testing instructions, I am willing.
